tools/esdlcmd/README.md

```diff
@@ -0,0 +1,485 @@
+| `@publishedBy` | No | string | Userid of the person publishing the binding |
+
+The `<em:ServiceBinding>` element is not required since it is possible to embed a complete `Binding` element hierarchy in the manifest.
+
+#### EsdlDefinition
+Recommended child of `<em:Manifest>` that enables ESDL definition-specific logic in the tool:
+
+- A `Definitions` element is output. All content is enclosed in a CDATA section.
+- The manifest `<em:Include>` element is recognized to import ESDL definitions:
+  - An included .ecm file is transformed into XML.
+  - An included .xml file is imported as-is.
+
+| Attribute   | Required? | Value  | Usage |
+| - | :-: | - | - |
+| N/A |
+
+The element is not required since it is possible to embed a complete `Definitions` element hierarchy in the manifest.
+
+#### Include
+Optional element that imports the contents of another file into the output, in place of the itself. The outcome of the import depends on the context in which this element is used. See [EsdlDefinition](#esdldefinition), [Scripts](@scripts), and [Transform](#transform) for more information.
+
+| Attribute   | Required? | Value  | Usage |
+| - | :-: | - | - |
+| `@file` | Yes | file path | Full or partial path to an external file to be imported. If a partial path is outside of the tool's working directory, the tool's command line must specify the appropriate root directory using either `-I` or `--include-path`. |
+
+>Any XSLTs or ESDL Scripts written inline in a manifest file will have XML escaping applied where required to generate valid XML. If an XLST contains any text content or markup that needs to be preserved as-is (no XML escaping applied) then be sure to use an `<em:Include>` operation. Included files are inserted into the output as-is, with the exception of preventing nested CDATA. If the included file will be inside a CDATA section on output, then any CDATA end markup in the file will be encoded as `]]]]><![CDATA[>` to prevent nested CDATA sections or a prematurely ending a CDATA section.
+>
+>For details on using XSLT to generate unescaped output, see this section of the specification: https://www.w3.org/TR/1999/REC-xslt-19991116#disable-output-escaping
+
+#### Scripts
+Optional repeatable element appearing within an `<em:ServiceBinding>` element that processes child elements and creates output expected for an ESDL binding:
+
+- The `<em:Scripts>` element is replaced on output with `<Scripts>`. Then all content is enclosed in a CDATA section after wrapping it with a new `Scripts` element. That new `<Scripts>` element contains namespaces declared by the input `<em:Scripts>` element. The input `<em:Scripts foo="..." xmlns:bar="..."><!-- content --></em:Scripts>` becomes `<Scripts><![CDATA[<Scripts xmlns:bar="..."><!-- content --></Scripts>]]></Scripts>`.
+- The manifest `<em:Include>` element is recognized to import scripts from external files. The entire file, minus leading and trailing whitespace, is imported.
```

Comment:
If sentence two is correct, perhaps a warning against including files that include XML declarations.

Reply:
added